Dockerfile.app-sre.rhel

```diff
@@ -0,0 +1,29 @@
+# This Dockerfile will replace Dockerfile.rhel once we migrate the service as part of APPSRE-2801
+
+# base image source https://github.com/rhdt/EL-Dockerfiles/blob/master/base/python3/Dockerfile
+
+FROM quay.io/app-sre/rhel-base-python3:latest
```

Comment:
@maorfr Is there a reason for not using ubi base image directly?

Reply:
if you want to migrate to UBI that is up to you. this migration is about moving you to App SRE standards (and off the openshiftio quay org and off centos).

Reply:
https://coreos.slack.com/archives/C01DKUK9P1A/p1609674941001400